scheduler/db/planmanager.py

```diff
@@ -28,15 +28,21 @@ def get_plans() -> List[SPlans]:
         This is to ensure that the plans are not corrupted after the
         lock is released.
         """
-        plans = deepcopy(db.read())
-        return plans
+        try:
+            plans = deepcopy(db.read())
+            return plans
+        except Exception as err:
```

Comment:
Can we be more specific as to the types of `Exception`s? PEP-8 says we should avoid using naked `except` or catch-all `except`. Is it possible that this could raise an `Error` which I don't think would be caught by an `except Exception`?